source/Modules/Devon4Net.Infrastructure.AWS.CDK/Options/Resources/DatabaseOptions.cs

```diff
@@ -36,5 +37,9 @@ public class DatabaseOptions
         public StorageType StorageType { get; set; }
         public string DatabaseEngineVersion { get; set; }
         public InstanceClass InstanceType { get; set; }
+        public int? EnhancedMonitoringIntervalSeconds { get; set; }
+        public string? MonitoringRoleName { get; set; }
```

Comment:
Please remove the ?, #nullable' annotations context is not set for the assembly where the class belongs to